2021-12-02 15:58:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:15 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:18 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:19 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:20 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 15:58:21 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 462, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:29 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:30 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:00:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 187, in m_time_check
    delta_check = datetime.timedelta(seconds=autocopy_delta_time)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:06 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:07 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:08 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:09 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:11 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:15 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:18 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:19 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:20 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:21 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:22 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:23 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:24 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:25 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:26 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:27 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:28 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:30 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:38 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:39 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:40 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:41 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:42 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:43 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:44 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:45 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:46 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:47 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:48 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:49 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:50 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:51 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:52 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:53 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:54 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:55 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:56 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:57 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:58 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:03:59 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:00 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:01 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:02 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:03 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:04 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 16:04:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 463, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:11:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:06 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:07 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:08 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:09 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:11 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:11:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:24 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:25 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:26 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:27 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:28 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:29 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:30 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:12:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:41 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:42 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:43 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:44 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:45 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:46 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:47 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:48 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:49 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:50 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:51 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:52 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:53 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:54 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:55 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:56 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:57 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:58 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:13:59 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:00 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:01 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:02 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:03 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:04 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:06 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:07 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:08 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:09 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:11 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:15 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:18 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:19 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:20 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:21 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:22 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:23 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:24 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:25 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:26 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:27 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:28 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:29 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:30 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:14:38 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:03 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:04 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:06 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:07 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:08 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:09 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:11 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:15 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:18 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:19 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:20 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:21 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:22 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:23 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:24 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:25 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:26 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:27 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:28 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:29 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:30 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:31 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:32 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:38 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:39 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:40 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:41 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:42 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:43 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:44 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:45 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:46 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:47 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:48 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:49 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:50 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:51 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:15:52 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 491, in autocopy_starter
    if file.black_list_check():
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 198, in black_list_check
    print('blacklist_check: '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:17:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:17:34 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:35 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:37 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:38 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:39 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:40 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:41 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:42 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:43 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:44 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:45 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:46 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:47 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:48 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:49 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:50 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:51 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:52 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:53 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:54 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:55 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:56 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:57 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:58 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:17:59 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:00 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:01 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:02 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:03 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:04 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:05 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:06 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:07 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:08 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:09 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:11 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:12 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:13 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:18:15 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 490, in autocopy_starter
    if file.m_time_check(autocopy_pause_time_check, autocopy_delta_time):
  File "C:\Users\Gekk0\PycharmProjects\360\BRIO_assistant\funcs_and_classes.py", line 188, in m_time_check
    pause_time = datetime.timedelta(seconds=autocopy_pause_time_check)
TypeError: unsupported type for timedelta seconds component: str

2021-12-02 17:20:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:23:17 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:23:45 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:25:21 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:25:59 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:28:03 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:28:20 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:28:40 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:28:58 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:29:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:29:36 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:30:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:30:55 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:31:33 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:32:10 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:32:48 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:33:25 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:33:59 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:49:58 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:51:28 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 17:52:16 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:04:04 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:17:14 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:19:23 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:19:51 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:25:46 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

2021-12-02 18:31:54 ERROR    Traceback (most recent call last):
  File "C:/Users/Gekk0/PycharmProjects/360/BRIO_assistant/interface.py", line 506, in autocopy_starter
    print('added to blacklist '+black_list)
TypeError: can only concatenate str (not "dict") to str

